_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,243 @@
+  that aims to improve the security of installing Bitcoin and Lightning nodes. It
+  uses the functional Linux distribution NixOS where upgrading is equivalent to
+  installing from scratch due to atomicity. Through minimalism, reproducibility,
+  compartmentalization and defense in depth nix-bitcoin aims to reduce the attack
+  surface of a Bitcoin and Lightning software stack with programs unaware of
+  other running processes unless they need to be. ([transcript][nixbitcoin
+  transcript], [audio][nixbitcoin audio])
+
+- **Taproot activation**: Eric Lombrozo, Luke Dashjr and Aaron van Wirdum
+  discussed the alternative [Taproot][topic taproot] activation proposals (see
+  [Newsletter #107][news107 taproot activation]) and gave their opinions in a
+  personal capacity on what lessons, if any, could be derived from activating the
+  SegWit soft fork.  Lombrozo and Dashjr both believe that the Taproot activation
+  process should not be drawn out and that all opposition, criticism or review of
+  the proposed code changes should be flushed out prior to starting any possible
+  activation process. As a result they support a single phase BIP 8 activation
+  method with parameters still to be determined. Community feedback on activation
+  proposals continues to be collected. ([transcript][activation transcript],
+  [video][activation video])
+
+- **Signet**: Kalle Alm and AJ Towns participated in a discussion on
+  [Signet][topic signet].  The design decisions of Signet were explored as well
+  as the mechanics of how testnet and regtest work. For more details see the
+  [_Default signet discussion_ news item](#default-signet-discussion).
+  ([transcript][signet transcript], [video][signet video])
+
+- **Bitcoin Core GUI meeting**: Anonymized participants including designers and
+  developers met to discuss the Bitcoin Core graphical user interface, its
+  current state, how it could be improved or revamped and the constraints that
+  any possible revamp would be subject to. For example, one change that has been
+  discussed in the past is moving from Qt Widgets to the Qt QML framework which
+  is more flexible and easily customized. ([transcript][bitcoin core gui
+  transcript]).
+
+- **Sydney meetup discussion** Ruben Somsen and Rusty Russell participated in a
+  discussion on statechains, upgrading LN channels and [lnprototest][]. Somsen
+  outlined how statechains (see [Newsletter #100][news100 statechains]) could
+  potentially be used to swap out your Lightning channel counterparty or in the
+  case of DLCs (see [Newsletter #81][news81 dlc]) swap out a portion of your
+  position without closing the channel. With various possible Lightning channel
+  upgrades on the horizon it was speculated which could use dynamic commitments
+  (see [Newsletter #108][news108 dynamic commitments]) and which would require
+  [splicing][topic splicing]. Finally, Russell explained how the Lightning
```

Comment:
This sentence is unclear. Not sure what "which" refers to.

Reply:
Some Lightning channel upgrades will be able to be upgraded using dynamic commitments and some won't, these will need splicing/onchain transactions. I don't know how to make this clearer. Maybe "which upgrades" rather than "which"?